### PR TITLE
chore(zh-cn): add language tags for fenced codes  (part 4)

### PR DESCRIPTION
**Pull request by yin1999**
### Description

add language tags for fenced codes  (part 4)

This is the final part which has enabled the markdown-lint rules.

### CODE REVIEW

File: files/zh-cn/.markdownlint.jsonc [hunk 1]
```diff
@@ -73,6 +74,26 @@
...
+      },
+      {
+        "name": "gfm-alert",
+        "message": "Use the GFM syntax: https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#备注、警告和标注",
+        "searchPattern": "/^ *> \\*\\*(Note|Warning|Callout):\\*\\*(?! [[{`_*])/gm",
+        "searchScope": "text",
+      },
+      {
+        "name": "bad-gfm-alert",
+        "message": "Use the correct GFM syntax: `> [!NOTE]`",
```
[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/.markdownlint.jsonc:92]: This is copy from en-US

[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/zh-cn/.markdownlint.jsonc:92]: @Josh-Cena shall we make the message a little bit more accurate - i did find the missing in the **expr**, but did not noticed that the message also misses something

[the remark this answers is not in the document]
Comment (Josh-Cena) [reply files/zh-cn/.markdownlint.jsonc:92]: I don't have strong opinions but the current one isn't too bad IMO. It's just an example of syntax.

[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/zh-cn/.markdownlint.jsonc:92]: > I don't have strong opinions but the current one isn't too bad IMO. It's just an example of syntax.

yeap its acceptable 🍉

File: files/zh-cn/web/css/_doublecolon_cue/index.md [hunk 2]
```diff
@@ -26,8 +26,10 @@ slug: Web/CSS/::cue
 
 ## 语法
 
-```
-::cue | ::cue( <selector> )
+```css
+::cue|::cue(<selector >) {
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_cue/index.md:30]:
````suggestion
```css-nolint
::cue | ::cue(<selector>) {
````

**Review by JasonLamv-t — APPROVED** [review]

**Pull request merged**